plugins/aws/tools/model_gen.py

```diff
@@ -276,7 +276,7 @@ def all_models() -> List[AwsModel]:
         # AwsResotoModel("get-resources", "items", "Resource", prefix="ApiGateway"),
         # AwsResotoModel("get-domain-names", "items", "DomainName", prefix="ApiGateway"),
         # AwsResotoModel("get-client-certificates", "items", "ClientCertificate", prefix="ApiGateway"),
-        AwsResotoModel("get-domain-names", "items", "DomainName", prefix="ApiGateway", prop_prefix="domain_")
+        # AwsResotoModel("get-domain-names", "items", "DomainName", prefix="ApiGateway", prop_prefix="domain_")
```

Comment:
consider to comment out

Reply:
ah, dang. will be more diligent upon the next service!